backend/src/modules/Announcements/repository/inMemory/InMemoryAnnouncement.repository.js

```diff
@@ -18,4 +18,29 @@ export class InMemoryAnnouncementRepository {
     }
     return announcement;
   }
+
+  async deleteOneByAnnouncementId(announcementId) {
+    const announcement = this.entities[announcementId];
+
+    if (!announcement) {
+      throw new NotFoundError('Announcement');
+    }
```

Comment:
Hej, baza danych z zalożenia nie powinna rzucać błędami. I teraz jak myślę, to nie powinna na zewnątrz repo błędami rzucać. 
Np. zobacz u siebie w pracy. Pytasz o idem z bazy, a tam się okazuje, ze nie istnieje. To nie rzucasz błędu, tylko zwracasz "undefined" albo pustą tablicę :)

Reply:
```suggestion
    if (!announcement) {
     return;
    }
```

lub

```suggestion
    if (!announcement) {
     return [];
    }
```

Reply:
Tutaj łamiesz zasadę "L" z SOLID

Reply:
done